<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestEditor</class>
 <widget class="QWidget" name="TestEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="WordingQuestionTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>371</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CurrentQuestionSpinBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="QuestionNumberLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер вопроса:</string>
    </property>
   </widget>
   <widget class="QLabel" name="WordingQuestionLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Формулировка вопроса</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="OneAnswerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="ManyAnswersButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="OneAnswer1Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="OneAnswer2Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="OneAnswer3Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="OneAnswer4Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="DetailedAnswerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="DetailedAnswerLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="ManyAnswers1Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>212</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ManyAnswers2Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ManyAnswers3Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ManyAnswers4Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstAnswerLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SecondAnswerLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ThirdAnswerLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FourthAnswerLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplyChangesButton">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>393</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить изменения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddNewQuestionButton">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>43</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить новый вопрос</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
